pkg/network/config.go

```diff
@@ -99,46 +99,3 @@ func NewDefaultConfig() *Config {
 		ClosedChannelSize:            500,
 	}
 }
-
-// EnabledKProbes returns a map of kprobes that are enabled per config settings.
-// This map does not include the probes used exclusively in the offset guessing process.
-func (c *Config) EnabledKProbes(pre410Kernel bool) map[KProbeName]struct{} {
```

Comment:
By the way, this was moved out of `config` because the linter was complaining the constants below were not defined for `windows`.